src/main/java/mandel/projectile/ProjectileGraph.java

```diff
@@ -0,0 +1,36 @@
+package mandel.projectile;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ProjectileGraph extends JComponent
+{
+    private Projectile projectile = new Projectile(0, 0);
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        g.translate(0, getHeight());
+        g.setColor(Color.BLUE);
+        g.fillOval((int) projectile.getInterceptX() / 2, (int) (-projectile.getPeakY()), 10, 10);
```

Comment:
the oval needs to be centered on the curve. Yours is not.

Reply:
Is it properly centered now?